azdo-task/README.md

```diff
@@ -1,10 +1,9 @@
-# DevcontainersCi
+# Dev Container Build and Run Task
```

Comment:
I like the way that the GitHub action includes the id in the title, maybe do the same here?

```suggestion
# Dev Container Build and Run Task (DevcontainersCi)
```